bluepyopt/ephys/morphologies.py

```diff
@@ -94,7 +94,7 @@ def instantiate(self, sim=None, icell=None):
         # TODO this is to get rid of stdout print of neuron
         # probably should be more intelligent here, and filter out the
         # lines we don't want
-        sim.neuron.h.hoc_stdout('/dev/null')
+        sim.neuron.h.hoc_stdout('null')
```

Comment:
Maybe NUL would work? (I don't have a windows machine)
It's a special file in windows that's somewhat equivalent to /dev/null, iirc.
https://technet.microsoft.com/library/Cc961816